src/components/Header/index.tsx

```diff
@@ -1,8 +1,22 @@
 import React from 'react'
-import { Title } from './styled'
+import { Theme, useThemeControl } from '../Theme'
+import { StyledToggle, Title, Wrapper } from './styled'
 
 const Header: React.FC = () => {
-	return <Title>Hello, World!</Title>
+	const [theme, setTheme] = useThemeControl()
+
+	return (
+		<Wrapper>
+			<Title>Hello, World!</Title>
+			<StyledToggle
```

Comment:
Awesome, reusable toggle! :heart: My only complain is that I liked it better when isRound=[true]. :sunny:

Reply:
Take this merely as an example. You can change it later to your liking. :wink:

Reply:
👍